posts/2024-02-15-Windows-7.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Updated baseline standards for Windows targets"
+author: Chris Denton on behalf of the Compiler Team
+---
+
+The minimum requirements for Tier 1 toolchains targeting Windows will increase with the 1.77 release (scheduled for March 21, 2024).
+Windows 10 will now be the minimum supported Rust version for the `*-pc-windows-*` targets.
```

Comment:
Windows 10 is not a Rust version. 😄 

Depending on how specific you want to be:

```suggestion
Windows 10 will now be the minimum supported Windows version for the `*-pc-windows-*` Rust targets.
```

or

```suggestion
Windows 10 will now be the minimum supported Windows version for the `*-pc-windows-*` targets.
```


or

```suggestion
Windows 10 will now be the minimum supported version for the `*-pc-windows-*` targets.
```